bin/email-failed-echo-bulky-payment

```diff
@@ -0,0 +1,87 @@
+#!/usr/bin/env perl
+
+use strict;
+use warnings;
+
+BEGIN {
+    use File::Basename qw(dirname);
+    use File::Spec;
+    my $d = dirname( File::Spec->rel2abs($0) );
+    require "$d/../setenv.pl";
+}
+
+use DateTime;
+use FixMyStreet::Cobrand;
+use FixMyStreet::DB;
+use FixMyStreet::Email;
+use mySociety::Locale;
+
+mySociety::Locale::gettext_domain('FixMyStreet');
```

Comment:
I'm not sure what this is here for, it shouldn't be necessary in any cron scripts? (It isn't in any others that I can see.) At most you might call `CronFns::language()` (that send-alerts/reports do), but other scripts that send emails don't, so I'd be interested to know what issue was leading to this.

Reply:
I was getting an error regarding an undefined $gettext in Locale.pm. `CronFns::language()` seems to work though.